jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -202,35 +199,6 @@ public static enum HookMode {
 		CONCURRENT_MODE
 	}
 
-	private static boolean loadNativeLibrary() {
-		if (!nativeLoaded) {
-			try {
-				AccessController.doPrivileged((PrivilegedAction<Void>) () -> {
-					System.loadLibrary("j9criu29"); //$NON-NLS-1$
```

Comment:
`System.loadLibrary("j9criu29"); ` is not needed, but we still need to call `setupJNIFieldIDsAndCRIUAPI()` to do a dynamic load of criu at class init so we know before hand if there are issues with the library when calling `isCRIUSupportEnabled`

Reply:
I think some of the checks were lost after doing the refactoring as nothing calls `isCRIUSupportEnabledAndNativeLoaded` anymore

Reply:
Added back `setupJNIFieldIDsAndCRIUAPI()` along with `loadNativeLibrary()` & `init()`.
Invoke it from `InternalCRIUSupport.checkpointJVM()` and `CRIUSupport.isCRIUSupportEnabled()`.